x-pack/plugins/cloud_security_posture/public/pages/compliance_dashboard/dashboard_sections/benchmarks_section.tsx

```diff
@@ -6,39 +6,54 @@
  */
 
 import React from 'react';
-import { EuiFlexItem, EuiFlexGroup, useEuiTheme, EuiTitle } from '@elastic/eui';
+import { EuiFlexGroup, EuiFlexItem, EuiTitle, useEuiTheme } from '@elastic/eui';
 import { FormattedMessage } from '@kbn/i18n-react';
 import { i18n } from '@kbn/i18n';
 import { CloudPostureScoreChart } from '../compliance_charts/cloud_posture_score_chart';
-import type { ComplianceDashboardData, Evaluation } from '../../../../common/types';
+import type {
+  ComplianceDashboardData,
+  Evaluation,
+  PosturePolicyTemplate,
+} from '../../../../common/types';
 import { RisksTable } from '../compliance_charts/risks_table';
-import { RULE_FAILED } from '../../../../common/constants';
+import { KSPM_POLICY_TEMPLATE, RULE_FAILED } from '../../../../common/constants';
 import { useNavigateFindings } from '../../../common/hooks/use_navigate_findings';
 import { ClusterDetailsBox } from './cluster_details_box';
-import { dashboardColumnsGrow } from './cloud_summary_section';
+import { dashboardColumnsGrow, getPolicyTemplateQuery } from './summary_section';
 
-export const CloudBenchmarksSection = ({
+export const BenchmarksSection = ({
   complianceData,
+  dashboardType,
 }: {
   complianceData: ComplianceDashboardData;
+  dashboardType: PosturePolicyTemplate;
 }) => {
   const { euiTheme } = useEuiTheme();
   const navToFindings = useNavigateFindings();
 
   const handleEvalCounterClick = (clusterId: string, evaluation: Evaluation) => {
-    navToFindings({ cluster_id: clusterId, 'result.evaluation': evaluation });
+    navToFindings({
+      cluster_id: clusterId,
+      'result.evaluation': evaluation,
+      ...getPolicyTemplateQuery(dashboardType),
+    });
   };
 
   const handleCellClick = (clusterId: string, ruleSection: string) => {
     navToFindings({
       cluster_id: clusterId,
       'rule.section': ruleSection,
       'result.evaluation': RULE_FAILED,
+      ...getPolicyTemplateQuery(dashboardType),
     });
   };
 
   const handleViewAllClick = (clusterId: string) => {
-    navToFindings({ cluster_id: clusterId, 'result.evaluation': RULE_FAILED });
+    navToFindings({
+      cluster_id: clusterId,
```

Comment:
is `cluster_id` relevant for CSPM? @CohenIdo

Reply:
there will be cluster ids and account ids in cspm, but until we actually know whats going on here, we rely on cluster id for now